yarn.lock

```diff
@@ -18787,6 +18800,16 @@ react-transition-group@^2.2.0, react-transition-group@^2.2.1:
     prop-types "^15.6.2"
     react-lifecycles-compat "^3.0.4"
 
+react-transition-group@^4.4.1:
```

Comment:
how did the yarn.lock entry for "react-transition-group": "^2.2.0" not get removed?

Reply:
storybook and `react-overlays` 

```
=> Found "react-transition-group@2.9.0"
info Has been hoisted to "react-transition-group"
info Reasons this module exists
   - Hoisted from "_project_#react-overlays#react-transition-group"
   - Hoisted from "_project_#@storybook#addon-knobs#react-select#react-transition-group"
=> Found "@react-spectrum/overlays#react-transition-group@4.4.1"
info This module exists because "_project_#@react-spectrum#overlays" depends on it.
```